src/Slim/Middleware/Session.php

```diff
@@ -79,6 +86,10 @@ protected function startSession()
     {
         $settings = $this->settings;
         $name = $settings['name'];
+        $handler = new $settings['handler'];
+        if (!$handler instanceof Handler) {
```

Comment:
`session_set_save_handler` will emit a warning if `$handler` is not a `SessionHandlerInterface`.
Sorry for users that have warnings disabled in development, but we need to get rid of this check.